test/fixtures/approvals/v3_non_readme_file.approved.json

```diff
@@ -0,0 +1,3 @@
+{
+  "error": "endpoint '/tracks/fake/fruit/strawberry/readme' not found."
+}
```

Comment:
I don't want an error state for this endpoint--I'd rather have an empty string for the README in the case where there it is missing (but it really shouldn't be missing, since we have linting for that).